### ISSUE TITLE
bilobed tongue

### ISSUE THREAD

**Issue by pnrobinson** [obophenotype/human-phenotype-ontology#6083]
**Preferred term label:**

**Synonyms**

Possibly identical with absent tongue tip

**Definition (free text, please give PubMed ID)**
See https://www.cell.com/ajhg/fulltext/S0002-9297(16)00009-4#supplementaryMaterial

**Parent term (use hpo.jax.org/app)**

**Diseases characterized by this term ? (e.g. Orphanet or OMIM number)**

**Your nano-attribution (ORCID)**

**Comment by pnrobinson** [comment]
In the article, they are using this as a synonym for bifid tongue (as documented in the supplement). I will add this as a synonym.
